x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -69,3 +70,97 @@ export const getSignalsTemplate = (index: string) => {
   };
   return template;
 };
+
+export const createSignalsFieldAliases = () => {
+  const fieldAliases: Record<string, unknown> = {};
+  Object.entries(aadFieldConversion).forEach(([key, value]) => {
+    fieldAliases[value] = {
+      type: 'alias',
+      path: key,
+    };
+  });
+  return fieldAliases;
+};
+
+export const getRbacRequiredFields = (spaceId: string) => {
+  return {
+    'kibana.space_ids': {
+      type: 'constant_keyword',
+      value: spaceId,
+    },
+    'kibana.consumers': {
```

Comment:
Can we use the const in the package to ensure we're all using the right field names?

Reply:
:+1: I updated the space IDs field to use the const names from the package, but the other fields are changing in https://github.com/elastic/kibana/pull/105096 so to avoid conflicts I just put the strings in as placeholders